eclair-rpc/src/test/scala/org/bitcoins/eclair/rpc/EclairRpcClientTest.scala

```diff
@@ -0,0 +1,100 @@
+package org.bitcoins.eclair.rpc
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import org.bitcoins.core.util.BitcoinSLogger
+import org.bitcoins.eclair.rpc.client.EclairRpcClient
+import org.bitcoins.eclair.rpc.json.ChannelResult
+import org.bitcoins.rpc.RpcUtil
+import org.bitcoins.rpc.client.BitcoindRpcClient
+import org.scalatest.{ Assertion, AsyncFlatSpec, BeforeAndAfterAll }
+
+import scala.concurrent.{ Await, Future }
+import scala.concurrent.duration.DurationInt
+
+class EclairRpcClientTest extends AsyncFlatSpec with BeforeAndAfterAll {
+
+  implicit val system = ActorSystem("EclairRpcClient")
+  implicit val m = ActorMaterializer.create(system)
+  implicit val ec = m.executionContext
+  implicit val bitcoinNp = EclairTestUtil.bitcoinNetwork
+
+  val logger = BitcoinSLogger.logger
+
+  val bitcoindInstance = EclairTestUtil.startedBitcoindInstance()
+  val bitcoindRpc = new BitcoindRpcClient(bitcoindInstance)
+
+  val e1Instance = EclairTestUtil.eclairInstance(bitcoindInstance)
+  val e2Instance = EclairTestUtil.eclairInstance(bitcoindInstance)
+
+  val client = new EclairRpcClient(e1Instance)
+  val otherClient = new EclairRpcClient(e2Instance)
+
+  override def beforeAll(): Unit = {
+    logger.info("Temp eclair directory created")
+    logger.info("Temp eclair directory created")
+
+    logger.debug(client.getDaemon.authCredentials.datadir.toString)
+
+    client.start()
+    otherClient.start()
+
+    RpcUtil.awaitCondition(
+      condition = () => client.isStarted(),
```

Comment:
are these`() =>` necessary? I think it's a bit more reader-friendly to drop them

Reply:
I this case they are not necessary, but in the more general case they are. I.e. if `client.isStarted()` took params you would need to use an explicit anonymous function.